### PR TITLE
Completed Section 3

### LINKED ISSUE #164: VnVPlan - Section 3
#### Subitems
- [ ] 3.1 Verification and Validation Team
- [ ] 3.2 SRS Verification Plan
- [ ] 3.3 Design Verification Plan
- [ ] 3.4 Verification and Validation Plan
- [ ] 3.5 Implementation Verification Plan
- [ ] 3.6 Automated Testing and Verification Tools
- [ ] 3.7 Software Validation Plan

### PR DESCRIPTION
**Pull request by KyleJMcMaster**
closes #164 

Please check checklists closely, and add anything I missed

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -142,95 +148,246 @@ \subsection{Relevant Documentation}
...
 
-\wss{What tools are you using for automated testing.  Likely a unit testing
-  framework and maybe a profiling tool, like ValGrind.  Other possible tools
-  include a static analyzer, make, continuous integration tools, test coverage
-  tools, etc.  Explain your plans for summarizing code coverage metrics.
-  Linters are another important class of tools.  For the programming language
-  you select, you should look at the available linters.  There may also be tools
-  that verify that coding standards have been respected, like flake9 for
-  Python.}
+Please refer to section 7.2.2 and 7.2.3 in the Development Plan found \href{https://github.com/OKKM-insights/OKKM.insights/blob/main/docs/DevelopmentPlan/DevelopmentPlan.pdf}{here}.
```
Comment (matpetro) [inline docs/VnVPlan/VnVPlan.tex:352]: I think we can expand on what is in the dev plan. There is no mention of actual testing frameworks such as pytest for python, maybe somethin like Jest or React test library for front end, mocking tools such as Mockito, whatever tools that would be necessary to test an ai model in some way. Also we can probably mention some code coverage tools for python and typescript too.

**Review by matpetro — CHANGES_REQUESTED** [review]
Overall looks good, not really sure what else you can say for these checklists. I think updating the dev plan/ mentioning in this doc some specific tools for testing could be beneficial, even if we have not confirmed them yet.

File: docs/VnVPlan/VnVPlan.tex (hunk 1 above)
Comment (KyleJMcMaster) [reply docs/VnVPlan/VnVPlan.tex:352]: That sounds great - I'll go through and add those. Are you okay to commit these changes now then?

Comment (KyleJMcMaster) [reply docs/VnVPlan/VnVPlan.tex:352]: Oh I see, you are suggesting I add the additional info in the VnV plan. I'll make the changes now and request a review when they're done.

Comment (matpetro) [reply docs/VnVPlan/VnVPlan.tex:352]: Use your discretion on whatever you think would be best, it wouldn't hurt to update both the devplan tex and Vnv plan tex under this pr. Honestly, just updating the dev plan may suffice

**Review by matpetro — APPROVED** [review]
Looks good, thanks

**Pull request merged**